<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string/>
  </property>
  <property name="dockOptions" >
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <property name="unifiedTitleAndToolBarOnMac" >
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout_3" >
    <item row="0" column="0" >
     <layout class="QVBoxLayout" name="verticalLayout" >
      <item>
       <layout class="QGridLayout" name="gridLayout" >
        <item row="0" column="0" >
         <widget class="QLineEdit" name="filepath" />
        </item>
        <item row="0" column="1" >
         <widget class="QPushButton" name="browseBtn" >
          <property name="text" >
           <string>&amp;Browse</string>
          </property>
          <property name="icon" >
           <iconset resource="../demo.qrc" >
            <normaloff>:/icons/document-open.png</normaloff>:/icons/document-open.png</iconset>
          </property>
          <property name="shortcut" >
           <string>Ctrl+B</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" >
        <property name="bottomMargin" >
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2" >
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="displayBtn" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Refresh</string>
          </property>
          <property name="icon" >
           <iconset resource="../demo.qrc" >
            <normaloff>:/icons/view-refresh.png</normaloff>:/icons/view-refresh.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_2" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="useCache" >
          <property name="text" >
           <string>Enable cache</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="autoDemo" >
          <property name="text" >
           <string>Auto demo</string>
          </property>
          <property name="shortcut" >
           <string>Ctrl+D</string>
          </property>
          <property name="checkable" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="timeoutDelay" >
          <property name="maximumSize" >
           <size>
            <width>30</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="inputMask" >
           <string/>
          </property>
          <property name="text" >
           <string>5</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>s.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line_3" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="runBtn" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Run</string>
          </property>
          <property name="icon" >
           <iconset resource="../demo.qrc" >
            <normaloff>:/icons/go-next.png</normaloff>:/icons/go-next.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="autoRun" >
          <property name="text" >
           <string>Auto Run</string>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0" >
     <widget class="QSplitter" name="splitter" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation" >
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="" >
       <layout class="QVBoxLayout" name="verticalLayout_2" >
        <property name="sizeConstraint" >
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <widget class="QListWidget" name="listWidget" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Maximum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="frame" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Maximum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QFrame" name="imginfo" >
          <property name="sizePolicy" >
           <sizepolicy vsizetype="Expanding" hsizetype="Maximum" >
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape" >
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow" >
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="" >
       <layout class="QGridLayout" name="gridLayout_2" >
        <property name="sizeConstraint" >
         <enum>QLayout::SetNoConstraint</enum>
        </property>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>729</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar" />
 </widget>
 <resources>
  <include location="../demo.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>filepath</sender>
   <signal>returnPressed()</signal>
   <receiver>displayBtn</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>213</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel" >
     <x>564</x>
     <y>64</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
